docs/en/examples/elementalarea_with_dataobject.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Using ElementalArea with DataObject
+summary: Learn how to add ElementalArea to your models and manage it in a ModalAdmin
+---
+
+# Using ElementalArea with DataObject
+
+## Creating Model and adding ElementalArea
+
+In this example the class BlogPost will be created, it will extend the DataObject class. Also a new BlogPost Admin section will be created in the Admin panel where CMS users can manage the BlogPosts.
+
+Let's look at an example:
+
+**app/src/Admins/BlogPostsAdmin.php**
+
+```php
+<?php
+
+namespace App\Admins;
+
+use App\Models\BlogPost;
+use SilverStripe\Admin\ModelAdmin;
+
+class BlogPostsAdmin extends ModelAdmin
+{
+
+    private static string $url_segment = 'blog-posts-admin';
+
+    private static string $menu_title = 'Blog Posts';
+
+    private static string $menu_icon_class = 'font-icon-block-banner';
+
+    private static array $managed_models = [
+        BlogPost::class,
+    ];
+}
+```
+
+`$managed_models` - this static variable connects BlogPostAdmin with BlogPost, which will be created in the following code.
+When you create a DataObject you should define the connection between your DataObject and ElementalArea by adding `$has_one` relationship.
+In this example BlogPost will have one ElementalArea.
```

Comment:
```suggestion
`$managed_models` - this static variable connects `BlogPostAdmin` with `BlogPost`, which will be created in the following code.
When you create a `DataObject` you should define the connection between your `DataObject` and `ElementalArea` by adding `$has_one` relationship.
In this example `BlogPost` will have one `ElementalArea`.
```

Reply:
Note that the new lines won't actually appear when the file is rendered - if your intention is for each of these to display on a new line, you'll need to add two spaces at the end of the lines (https://www.markdownguide.org/basic-syntax#line-breaks).
I won't repeat this comment elsewhere but it applies to all such situations in the document.